fastify.js

```diff
@@ -502,6 +502,9 @@ function fastify (options) {
     } else if (name === 'onReady') {
       this[kHooks].validate(name, fn)
       this[kHooks].add(name, fn)
+    } else if (name === 'onRoute') {
```

Comment:
I need to move the setup for the `onRoute` hooks due to the fact that was made for being async, in order to achieve register routes sync, the hooks should also be set in the same way. Please let me know your thoughts